docs/features.md

```diff
@@ -0,0 +1,8 @@
+### Features
+
+| k8s | Features |
```

Comment:
Let us change this table as follows:

| Feature | Minimum K8s version |  Minimum ZFS Local PV CSI driver version |

Reply:
should we list alpha release also in "Minimum ZFS Local PV CSI driver version"?

Reply:
thats a good point. I think it doesn't make sense. but probably will help provide clarity in case someone wants to upgrade from their current version. 

Also, how do we compare to this list here: https://github.com/container-storage-interface/spec/blob/master/lib/go/csi/csi.pb.go#L242

Reply:
updated the details with the ZFS driver version.

>Also, how do we compare to this list here: 

That is the list of capabilities. This may not transalate to features, for example PUBLISH_UNPUBLISH is a capability which driver can decide whether it needs that or not. It may not convert as feature for the user. But it may be needed to provide the features like provisioning/deprovisioning of volumes.